theme_v2_json_structure.py

```diff
@@ -0,0 +1,366 @@
+        "meta_description": {
+            "ar": "المعهد العقاري",
+            "en": "SREI excellence"
+        },
+    },
+
+    "footer": {
+        "sections": [  #--  API key: footer_sections
+             {
+                 "key": "option_1",
+                 "link_title": {
+                     "ar": "",
+                     "en": ""
+                 },
+                 "page": "courses",  # or home, about_us, pages_custom_page_1, ... or https://
+                 #-- Display one of the list of pages,
+                 "new_tab": True,
+             },
+             {
+                 "key": "option_2",
+                 "link_title": {
+                     "ar": "",
+                     "en": ""
+                 },
+                 "page": "about_us",  # or home, about_us, pages_custom_page_1, ... or https://
+                 #-- Display one of the list of pages,
+                 "new_tab": True,
+             },
+             {
+                 "key": "option_3",
+                 "link_title": {
+                     "ar": "",
+                     "en": ""
+                 },
+                 "page": "pages_custom_page_1",  # or home, about_us, pages_custom_page_1, ... or https://
+                 #-- Display one of the list of pages,
+                 "new_tab": True,
+             },
+        ],
+        "social_media_links": [  #--  API key: footer_social_media_links
+            {
+                "provider": "facebook",
+                "value": "url"
+            },
+            {
+                "provider": "x",
+                "value": "url"
+            }
+        ]
+    },
+
+    "header": {
+        "sections": [  #--  API key: header_sections
+            {
+                "key": "option_1",
+                "link_title": {
+                    "ar": "",
+                    "en": ""
+                },
+                "page": "courses",  # or home, about_us, pages_custom_page_1, ... or https://
+                #-- Display one of the list of pages,
+                "new_tab": True,
+            },
+            {
+                "key": "option_2",
+                "link_title": {
+                    "ar": "",
+                    "en": ""
+                },
+                "page": "about_us",  # or home, about_us, pages_custom_page_1, ... or https://
+                #-- Display one of the list of pages,
+                "new_tab": True,
+            },
+            {
+                "key": "option_3",
+                "link_title": {
+                    "ar": "",
+                    "en": ""
+                },
+                "page": "pages_custom_page_1",  # or home, about_us, pages_custom_page_1, ... or https://
+                #-- Display one of the list of pages,
+                "new_tab": True,
+            },
+        ],
+        "combined_login": True,  #--  API key: header_combined_login
+    },
+}
+
+root_settings = {
+    "logo_image_url": "url",  #--  API key: logo_url
```

Comment:
right?

```suggestion
    "LOGO_URL": "url",  #--  API key: logo_url
```

Reply:
`LOGO_URL` is the global fall-back. The one per tenant config is `logo_image_url`. Working fine in my local

Reply:
Thanks @shadinaif!!